MyQueue.java

```diff
@@ -0,0 +1,36 @@
+
+//o(n) time for add and o(1) tiem for other functions
+//o(n) space
+
+//to add element recursive;ly go to the  bottom of stack and ad the element
+import java.util.Stack;
```

Comment:
Good Work! Please try if it is possible to make the add operation as O(1).

Reply:
HI, I have made the set operations as o(1) using the double hashing technique. Do we need to make o(1) for queue using stack as well?